man/systemd-socket-proxyd.xml

```diff
@@ -85,6 +85,12 @@
     <variablelist>
       <xi:include href="standard-options.xml" xpointer="help" />
       <xi:include href="standard-options.xml" xpointer="version" />
+      <varlistentry>
+        <term><option>--connections-max</option></term>
```

Comment:
Hmm, so, the respective options in .socket units is called `MaxConnections=`, hence I'd propose we call this option `--max-connections=`, too, just in order to stick to one nomenclature.

Also, we usually put a "=" at the end of command line switches that expect an argument in the docs, to clarify that they expect an argument.